packages/blue-sdk-ethers-liquidation/src/addresses.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  Address,
+  ChainAddresses,
+  ChainId,
+  addresses,
+} from "@morpho-org/blue-sdk";
+
+declare module "@morpho-org/blue-sdk" {
+  interface ChainAddresses {
+    timeBoundedUrd?: Address;
+    marketRewardsProgramRegistry?: Address;
+
+    weEth?: Address;
+    ezEth?: Address;
+    rsEth?: Address;
+    usdE?: Address;
+    sUsdE?: Address;
+    "usd0++"?: Address;
+    "usd0usd0++"?: Address;
+    verUsdc?: Address;
+    rsweth?: Address;
+    bsdEth?: Address;
+    re7Weth?: Address;
+    "PT-USDe-25JUL2024"?: Address;
+    "PT-ezETH-26DEC2024"?: Address;
+    "PT-sUSDE-24OCT2024"?: Address;
+    "PT-weETH-26DEC2024"?: Address;
+    "PT-weETH-27JUN2024"?: Address;
```

Comment:
Would be very useful to have the tag `pendle` in assets in API

Reply:
Will create a ticket for that